### PR TITLE
telegram bot sources added

### PR DESCRIPTION
**Pull request by Art-T**
Сорцы для бота, как и просили.

### CODE REVIEW

File: telegram-bot/.idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (maiksaray) [inline telegram-bot/.idea/compiler.xml]: Can we update gitignore so that we don't have these in repo?

File: telegram-bot/telegram-java-bot.iml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (maiksaray) [inline telegram-bot/telegram-java-bot.iml]: Do we really need both pom and iml?

**Comment by maiksaray** [comment]
Can you resolve conflicts pls?

**Comment by Art-T** [comment]
Conflict resolved, please check.

**Pull request merged**